.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,7 @@ Please describe the tests that you ran to verify your changes.
 -   [ ] I have performed a self-review of my own code
 -   [ ] I have added tests to cover my changes
 -   [ ] I have made corresponding changes to the documentation
+-   [ ] Optional: I created a RFC (Only for complex and/or impactful changes as defined in the [Contributing Guidelines](https://github.com/Farfetch/)kafkaflow/blob/master/CONTRIBUTING.md)
```

Comment:
It could be worth mentioning changes that will required to modify the API, what do you think?

Reply:
What I was going for, is to leave the description here as simple as possible, and the describe in the contributing.md the more specific cases where a RFC is expected ([this paragraph](https://github.com/Farfetch/kafkaflow/pull/442/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R14))

Reply:
Fair enough, no problem for me